app/models/shipit/deploy_spec.rb

```diff
@@ -40,6 +40,10 @@ def config(*keys, &default)
       end
     end
 
+    def config_empty?
```

Comment:
Sorry to be nitpicking, but it's not really what I expected.

I'd like something that communicate the state in term of business rules. Something that explains that this deploy spec is imcomplete, or missing, or something like that.

`blank?` could work as well, but it might have some unexpected consequences.

Reply:
I understand what you mean. I guess the way I saw it, I didn't want to imply that the whole DeploySpec was empty/blank/etc. I am okay with 'blank?'.  @benlangfeld Do you have any input?

Reply:
I'd say try `blank?`. CI should tell us if there's fallouts.